config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
```

Comment:
- [ ] Please add users#index as the root route as on the wireframe. You can consider using the below code.
```Ruby
root "users#index"